bundles/org.eclipse.passage.ldc.pde.ui.templates/templates/LicensedE4Product/$pluginId$.product

```diff
@@ -21,11 +21,6 @@
       <plugin id="com.fasterxml.jackson.core.jackson-core"/>
       <plugin id="com.fasterxml.jackson.core.jackson-databind"/>
       <plugin id="com.github.oshi.oshi-core"/>
-      <plugin id="com.ibm.icu"/>
-      <plugin id="com.sun.jna"/>
```

Comment:
In case any Plugin listed explicitly requires them (transitively) via Require-Bundle or Import-Package, then at least Tycho will install it any way into the product.
I'm not sure about the PDE product export at the moment, IIRC that does not include transitive dependencies, but one has to check that. 
Is the main use-case for those template products to be exported from the Product-Editor or build by Tycho?

Reply:
The main use case is to demonstrate how to start using Passage for an Eclipse RCP:
* create by template from PDE wizard
* open .product manifest
* launch

PDE should be able to fulfill the .launch definition and run the created product

Reply:
> PDE should be able to fulfill the .launch definition and run the created product

Yes when launching a product PDE is capable to include required transitive dependency into the launch automatically.
So in that use-case should work fine.